### ISSUE TITLE
After the search, the mouse is not focused on the page, such as Google

### ISSUE THREAD

**Issue by barrer** [alvinhochun/chrome-bookmark-search#7]
After the search, the mouse is not focused on the page, such as Google.
step:
bm -> google (match: www.google.com) -> enter

**Comment by alon91** [comment]
Yes -- this is undesirable behavior for sure. Also wish that the first result was the best-matching search result rather than an option to search for the string in the Chrome book manager (which is also broken).

Would appreciate suggestions for a similar extension that is more polished (or perhaps being actively developed -- this one is from 2013).

**Comment by alvinhochun** [comment]
So here's the thing, I don't think there is a proper way to un-focus the omnibox when using the `chrome.tabs.update` function to update the current tab. It has been like this for a long time. There appears to be some [hackish way that might've worked in the past][1], but they are prone to breaking and don't work properly. I do agree that the page should be focused after navigating, but it seems to be a missing function, unless I missed it from the API docs.

If you choose to open the links in a new tab (either by pressing Alt+Enter or changing it in the options), you don't get this problem.

[1]: https://stackoverflow.com/questions/5182695/chrome-extension-focus-on-the-page-instead-of-omnibox-after-selected-tab-is-upd

**Comment by alvinhochun** [comment]
@alon91 If you have separate requests, you could've opened a new issue instead of stuffing stuff in an unrelated issue to help keep things more organized. (Doesn't matter now that you've already commented here.)

This is a simple extension which just searches through the bookmarks in one way or another, it doesn't really need much polish. It did and still does what it aims to do. What usually happens is that Chrome changes some of its internal workings and break stuff. For instance I've had to change the bookmark page link for a few times in the past already.

I made this extension in 2011 just for fun, but it turns out to be useful to some users so I tried to keep up with the changes, at least when I was still studying and had more free time. I will try to get a new version out in a few days that fixes and adds a few things, but don't hold your breath.
